src/routes/docs/advanced/security/ccpa/+page.markdoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: article
+title: CCPA
+description: Protecting your and your users' data privacy is a priority at Appwrite. Learn about Appwrite's compliance with the California Consumer Privacy Act (CCPA).
+---
+
+Appwrite is compliant with the California Consumer Privacy Act (CCPA). The CCPA is a privacy law that gives California residents more control over their personal information, helping ensure their data privacy rights.
+
+To confirm Appwrite's compliance with the CCPA, we have ensured the following rights for users:
+
+- **Right to know:** Appwrite users can request information about the personal data that is collected, shared, or sold.
+- **Right to delete:** Users can request that Appwrite delete their personal data, with certain exceptions.
+- **Right to opt-out:** Users can opt out of the sale or sharing of their personal data with third parties.
+- **Right to non-discrimination:** Appwrite does not discriminate against users who exercise their CCPA rights.
+- **Right to correct:** Users can correct inaccurate personal information held by Appwrite.
+- **Right to limit:** Users can limit how Appwrite uses and shares their sensitive personal information.
+
+Some of the measures that Appwrite has taken for compliance include:
+
+- CCPA references in the DPA for both customers and vendors.
+- A detailed summary of data subject rights under CCPA, with a commitment to assist customers in compliance.
+- Employee training on handling privacy-related inquiries and Appwrite's adherence to CCPA requirements.
+- Updates to Appwrite's data retention and deletion policies.
+- Revisions to the data breach and incident response policies.
```

Comment:
At the end it might be worth mentioning that these are measures done by us. If they develop app that stores personal information in Appwrite Databases, they need to also comply with CCPA for their app to be considered complient.

Reply:
@LauraDuRy What do you think?

Reply:
Let's add somethng like this:
`Please note that while Appwrite Cloud serves as a CCPA-compliant platform to handle data, it is the responsibility of developers to ensure that their application is also compliant with CCPA regulations.`

Reply:
Done